_posts/2022-01-20-reached-4000-users.md

```diff
@@ -0,0 +1,21 @@
+---
+site: [pasteur, freiburg, erasmusmc, elixir-it, belgium, genouest]
+title: 'Usegalaxy.eu surpassed 40.000 registered users'
+tags: [galaxy, support, society, report]
+supporters:
+- galaxy-europe
+---
+
+We are of celebration! The last November usegalaxy.eu reached 40.000 registered users! Over the last year, the number of registered users has almost doubled, from 25,000 to more than 41,000 by January 2022. As we can see in figure 1, the growth in the number of new registrants shows an almost exponential trend since 2013.
+    
+![Total number of registered](/assets/media/reached_40000/total_users_galaxyeu.png)
+    
+*Figure 1. Total number of users registered in usegalaxy.eu since 2013.*
+
+As indicated in figure 2, this trend is also reflected in the monthly increase in users. The highest number of registered users in a single month was in October 2011, when 2.449 users joined the Galaxy Europe community.
+
+![Number of new users montly](/assets/media/reached_40000/users_galaxyeu.png)
+    
+*Figure 2. Number of new users registered monthly since 2013.*
+
+From Galaxy Europe, we thank you for the trust you place in us, and hope that the only exponential growth curve we have to report in the coming year will be the unstoppable growth of the Galaxy community.
```

Comment:
😆 🐛

Reply:
I can remove the sentence if you consider that it doesn't fit at all.

Reply:
no no, I think it fits and it's a nice analogy to close the post :) I actually thought that it was merged already, sorry that I missed that. Will merge then, thanks a lot for the plots and the text!!